scripts/ci/validate-cabal-files.sh

```diff
@@ -37,14 +37,14 @@ stack2cabal
 for file in "${files_to_verify[@]}"; do
     set +e
     diff_res=$(diff "$file" "${file}_backup")
+    set -e
     if [ "$diff_res" != "" ]; then
```

Comment:
Isn't it more proper to check the exit code here?

Reply:
:monkahmm: